Formula/tinker.rb

```diff
@@ -59,7 +59,7 @@ def ruby_version
   #
   # @return [Object] Reference to metadata object loaded from a gem.
   def metadata
-    @metadata ||= YAML.load(`tar -xOf #{tinker_gem_path} metadata.gz | gzip -dc`)
+    @metadata ||= YAML.load(`tar -xOf #{tinker_gem_path} metadata.gz | gzip -dc`, permitted_classes: [Gem::Specification, Gem::Version, Gem::Dependency, Gem::Requirement, Time, Symbol])
```

Comment:
I was able to get the metadata by using this standard library class. 
```ruby
require 'rubygems/package'

@metadata ||= Gem::Package.new(tinker_gem_path).spec
```

Reply:
Nice! It worked for me also.